app/views/jobs/index.html.erb

```diff
@@ -19,9 +20,11 @@
           </tr>
 
         <% @jobs.each do |job| %>
+          <% agent = Agent.find_by_id(YAML.load(job.handler).args[0]) %>
```

Comment:
Maybe this could become a method of Delayed::Job defined in `config/initializers/delayed_job.rb`.